_dev/src/components/smart-shopping-campaign/smart-shopping-campaign-table-list.vue

```diff
@@ -4,40 +4,29 @@
       v-if="!inNeedOfConfiguration"
       @openPopinRemarketingTag="remarketingTagPopin"
     />
-    <h3 class="order-2 order-md-1 ps_gs-fz-20 font-weight-600">
-      {{ $t('smartShoppingCampaignList.tableTitle') }}
-    </h3>
     <div
       v-if="!inNeedOfConfiguration"
       class="d-flex flex-wrap flex-md-nowrap justify-content-between mb-md-3 rounded-top"
+    />
```

Comment:
Something needs to be checked here. I see now that we close immediately the `<div>` tag. Is there a `</div>` to remove farther in the file?

Reply:
this is an ancient div, it's not used anymore. I'm removing it